k8s/wordpress/apptest/tester/tester.sh

```diff
@@ -18,5 +18,8 @@ set -xeo pipefail
 shopt -s nullglob
 
 for test in /tests/*; do
-  testrunner -logtostderr "--test_spec=${test}"
+  testspec="$(mktemp XXXXXXXX.yaml)"
```

Comment:
Please rename `xxxxxx` to something more descriptive.

Reply:
`mktemp` will replace `XXXXXXXX` with a random string. I would like to have random string instead of static file name.